blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -62,7 +62,8 @@ export class RunDetailsPipeline extends Component {
             }
         }
 
-        this.listener.sse = sseConnection.subscribe('pipeline', this._onSseEvent);
+        this.listener.ssePipeline = sseConnection.subscribe('pipeline', this._onSseEvent);
+        this.listener.sseJob = sseConnection.subscribe('job', this._onSseEvent);
```

Comment:
It's probably ok, but I just don't like the idea of registering a single listener function instance to multiple things. It's being a bit too clever for my liking and a recipe for future bugs.